### ISSUE TITLE
Sidebar mobile hamburger menu button is not clickable

### ISSUE THREAD

**Issue by crtr0** [tailwindui/issues#222]
**What component (if applicable)**
- URL for category: https://tailwindui.com/components/application-ui/application-shells/sidebar#component-42fe67e37320d5e72cdacb8a2e6d12a1
- Component name: Light sidebar with light background

**Describe the bug**

When using the code as provided, the hamburger button is not clickable. 

When I look at the (working) code on the website, I notice this line of code:

`<div x-show="sidebarOpen" class="md:hidden" style="display: none;">`

The corresponding line in the code sample is:

`<div class="md:hidden">`

The `style="display:none;"` gets removed and re-added as the mobile menu is open and closed, so I suspect there are Transition rules controlling this behavior, but they are not documented in the code sample.

**To Reproduce**
1. Copy-and-paste sample page
2. Convert to React, add Transition tags
3. Render page at mobile viewport width

**Expected behavior**

Clicking the hamburger menu opens the menu.

**Screenshots**

**Browser/Device (if applicable)**
 - OS: MacOS
 - Browser: Chrome
 - Version: 83

**Additional context**

**Comment by francodalessio** [comment]
@crtr0 Hi! Can you please share how you made it work? Or even the full working code, that would be awesome. I'm driving myself crazy, I've tried a bunch of things but I'm always breaking something...

**Comment by crtr0** [comment]
@francodalessio Sure! It involves the need to nest <Transition> tags. Here's a link to some source, I hope it helps!

https://github.com/FizbuzHQ/fizbuz/blob/master/src/layouts/AppLayout.tsx#L138

**Comment by francodalessio** [comment]
@crtr0 Indeed, I was missing one Transition tag. Thank you very very much for the quick response!

**Comment by blueslurpee** [comment]
@crtr0 Thanks so much for posting this, saved me
